README.md

```diff
@@ -84,7 +76,27 @@ The following optional config parameters are available for the Odrive:
 }
 ```
 
-## Sample Viam CAN Config with 2 Odrives and odrive_config_files
+The following optional configuration attributes are available for the ODrive:
+
+1. `odrive_config_file`: path to a separate JSON file containing your ODrive's native configuration (string).
+    * Only add this attribute if you want to reconfigure your ODrive natively each time the motor is initialized on the robot.
+       * Extract your configurations from your ODrive. To do so you must have `odrivetool` installed. See the [ODrive documentation](https://docs.odriverobotics.com/v/latest/odrivetool.html) for instructions on how to do this.
+       * After installing `odrivetool` run `odrivetool backup-config config.json` to extract your configs to a file called `config.json`. See [ODrive instructions](https://docs.odriverobotics.com/v/latest/odrivetool.html#configuration-backup) for more info.
+       * `iq_msg_rate_ms` in the config defaults to `0`. Set this to or around `100` to use the [motor API's `SetPower` method](https://docs.viam.com/components/motor/#setpower).
+    * (For `"canbus"` connection) If you add an `odrive_config_file`, you will have to leave the ODrive plugged in to the USB port in addition to wiring the CANH and CANL pins.
+    * An alternative to adding an `odrive_config_file` is running the command `odrivetool restore-config /path/to/config.json` in your terminal.
+    * See sample-configs/config.json for a sample ODrive config file.
+
+2. `serial_number`: serial number of the ODrive (string).
```

Comment:
@martha-johnston, what do you think of breaking these config instructions out out by model?

Reply:
that sounds good to me. I'm not sure what the best way to do it would be since there is some overlap in the serial and can bus instructions, but if we don't care about it being repetitive then yeah that works

Reply:
@sguequierre do you want to do this or keep documentation in its current form for now?

Reply:
Hmm-- I agree with Martha that the overlap could be too much here. I could try converting at least some of this to a table like in https://github.com/viamrobotics/docs/pull/1446 to simplify the attribute info?